portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** A user-inputted comment. */
+public final class Comment {
```

Comment:
(not sure if my other comment went through)

You don't have to do it here because this is a simpe case, but usually this kind of data-type class is well served with the builder design pattern. 

See:
https://refactoring.guru/design-patterns/builder